### PR TITLE
Continuing CCDA Saga

### PR DESCRIPTION
**Pull request by sjpadgett**
- fix encounter history user code types
- add self care observation functional status

### CODE REVIEW

File: ccdaservice/oe-blue-button-generate/lib/entryLevel/functionalStatusEntryLevel.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ exports.mentalStatusObservation = {
             attributes: {
                 code: "373930000",
                 codeSystem: "2.16.840.1.113883.6.96",
-                codeSystemName: "SNOMED-CT",
+                codeSystemName: "SNOMED CT",
```
Comment (stephenwaite) [inline ccdaservice/oe-blue-button-generate/lib/entryLevel/functionalStatusEntryLevel.js:24]: hah, you need both spellings of SNOMED w and w/o the dash? and then this entry only wants no dash?

**Comment by bradymiller** [comment]
2DV

**Review by bradymiller — APPROVED** [review]

**Comment by sjpadgett** [comment]
image

**Comment by sjpadgett** [comment]
Unfortunately I can't seem to help myself. Encounters History was bugging me and needs to be a container fluid form with form preview tooltips auto sizing. If anyone has a problem or thinks otherwise, please let me know and I'll revert.

image

Fixed to work with dark themes:

image

**Comment by stephenwaite** [comment]
make-it-look-easy

**Comment by sjpadgett** [comment]
New HTML document styling next commit:

image

**Comment by adunsulag** [comment]
It looks very nice @sjpadgett Great job!

**Comment by sjpadgett** [comment]
Thanks Stephen! I'm going to merge what I have here w/o new html document style. Will do a separate P.R that will cover a restructure of the project. I finished consolidation of the sections templating into separate exports and removed the old distribution SPA.

**Comment by bradymiller** [comment]
tenor99

**Pull request merged**